src/components/organisms/ChatSection/ChatSection.tsx

```diff
@@ -22,12 +49,20 @@ export const ChatSection = () => {
           variant="filled"
           multiline
           maxRows="6"
+          onKeyDown={(e) => {
```

Comment:
@APrzybylowska - "e" to error czy event czy coś jeszcze?

Reply:
Destrukturyzacja?

Reply:
Plus jakiego typu jest "e" - w TS mimo, że nie podkreśla to powinniśmy otypować

Reply:
event. ok, poprawię